package.json

```diff
@@ -14,8 +16,8 @@
     "google-auth-library": "^0.12.0",
     "googleapis": "^25.0.0",
     "newrelic": "^4.9.0",
-    "probot": "^5.0.0",
-    "probot-scheduler": "^1.2.0",
+    "probot": "^9.5.3",
```

Comment:
Why not 9.11.3, the latest version? (https://github.com/probot/probot/releases)

Reply:
Hmmm
How I did the update was I created a new probot app and used the version I found in the package.json generated by probot.

Updated.